en/docs/deploy/eks.md

```diff
@@ -12,11 +12,37 @@
 
     4.  Set up an [EKS cluster](https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html) on AWS cloud, if you do not have one set up already. 
     
-        !!! info 
-            After creating the EKS cluster, make sure you install the **aws-iam-authenticator** and create a **kubeconfig** by following the Amazon EKS guide mentioned above, to authenticate and communicate with your cluster respectively. After the communication is established, **add worker nodes** to your cluster before proceeding with the deployment.
-
-    5.  Install [NGINX Ingress Controller](https://kubernetes.github.io/ingress-nginx/deploy/) **version-nginx-0.22.0**. You can get the raw file for the recommended version [here](https://github.com/kubernetes/ingress-nginx/releases/tag/nginx-0.22.0). Make sure you install the mandatory.yaml file along with the service-xx.yaml and patch-configmap-xx.yaml files you installed for either layer 4 or layer 7.  
-
+        !!! note 
+            Make sure you have set up the following aspects properly before you set up the cluster. For more information, see [the amazon EKS getting started guide](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-console.html#eks-prereqs)
+
+			-	The EKS Service Role
+			-	The EKS Cluster VPC
+			-	AWS CLI version 2 
+				!!! note 
+					AWS CLI version 1 works fine too. However, you need to make sure that you have python3. AWS CLI version 2 is independant of the python version you are using. 
+	
+	5.	Launch a managed node to get started with your instance profile. For instructions to do this, see [here](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-console.html#eks-launch-workers).
```

Comment:
For instructions, see the [AWS user guide](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-console.html#eks-launch-workers).